src/stories/Library/Modals/modal-cta/modal-cta.scss

```diff
@@ -9,12 +9,22 @@
 }
 
 .modal-cta__buttons {
-  display: grid;
-  justify-content: center;
+  display: flex;
```

Comment:
I don't understand this part of the change. Shouldn't it give the same result as:
```
display: grid;
justify-content: center;
```

Reply:
before, the button had ':0 96px' padding. In this case what you are saying would work. 

Since there is no padding on mobile now, justify-content would squeeze it together. 

What i could've done i think, is use justify-content: stretch instead and that would've done the same thing. 

Now i just used flex instead, but probably would've been faster to just use justify-content: stretch